components/layout/side-bar.jsx

```diff
@@ -32,7 +32,12 @@ const SideBar = ({
           tag="button"
           onClick={() => dispatch({ type: 'toggle_sidebar' })}
         >
-          <CloseIcon aria-label="Close sidebar" />
+          <Icon
+            className={styles.icon}
+            src="#close"
+            aria-label="Close sidebar"
+            aria-hidden
```

Comment:
I do not see really combination of `aria-hidden` and `aria-label`, these are mutually excluding.

The similar comment to `alt`. It looks silly but let's leave it.

```suggestion
            alt="Close icon"
            aria-label="Close sidebar"
```